features/docs/gegeven-stap-definities-ruwe-gegevens.feature

```diff
@@ -0,0 +1,175 @@
+      En heeft persoon 'P1' de volgende rijen in tabel 'lo3_pl_persoon'
+        | pl_id | persoon_type | stapel_nr | volg_nr | burger_service_nr | voor_naam | geslachts_naam |
+        | P1    | P            |         0 |       0 |         000000012 |           | P1             |
+        | P1    |            1 |         0 |       0 |         000000024 | Helena    | Hanssen        |
+        | P1    | R            |         0 |       0 |         000000036 | Herman    | Hapert         |
+
+  Regel: {object soort} is {gewijzigd of gecorrigeerd} naar de volgende gegevens
+
+    @integratie
+    Abstract Scenario: <object soort wijziging> is <soort wijziging> naar de volgende gegevens
+      Gegeven de persoon 'P1' met burgerservicenummer '000000012'
+      * heeft <object soort toevoeging> met de volgende gegevens
+        | naam                  | waarde  |
+        | voornamen (02.10)     | Helena  |
+        | geslachtsnaam (02.40) | Hanssen |
+      En <object soort wijziging> is <soort wijziging> naar de volgende gegevens
+        | naam                        | waarde    |
+        | burgerservicenummer (01.20) | 000000024 |
+        | voornamen (02.10)           | Herman    |
+        | geslachtsnaam (02.40)       | Hanssen   |
+      Als de sql statements gegenereerd uit de gegeven stappen zijn uitgevoerd
+      Dan heeft persoon 'P1' de volgende rij in tabel 'lo3_pl'
+        | pl_id | geheim_ind |
+        | P1    |          0 |
+      En heeft persoon 'P1' de volgende rijen in tabel 'lo3_pl_persoon'
+        | pl_id | persoon_type   | stapel_nr | volg_nr | burger_service_nr | voor_naam | geslachts_naam | onjuist_ind |
+        | P1    | P              |         0 |       0 |         000000012 |           | P1             |             |
+        | P1    | <persoon type> |         0 |       1 |                   | Helena    | Hanssen        | <onjuist>   |
+        | P1    | <persoon type> |         0 |       0 |         000000024 | Herman    | Hanssen        |             |
+
+      Voorbeelden:
+        | object soort toevoeging | object soort wijziging | persoon type | soort wijziging | onjuist |
+        | een ouder 1             | ouder 1                |            1 | gewijzigd       |         |
+        | een ouder 1             | ouder 1                |            1 | gecorrigeerd    | O       |
+        | een ouder 2             | ouder 2                |            2 | gewijzigd       |         |
+        | een ouder 2             | ouder 2                |            2 | gecorrigeerd    | O       |
+        | een partner             | de partner             | R            | gewijzigd       |         |
+        | een partner             | de partner             | R            | gecorrigeerd    | O       |
+        | een kind                | het kind               | K            | gewijzigd       |         |
+        | een kind                | het kind               | K            | gecorrigeerd    | O       |
```

Comment:
hoe wordt bij een persoon met meerdere kinderen een specifiek kind gecorrigeerd/gewijzigd?

Reply:
de functie zoekt het kind op burgerservicenummer. Wanneer dat kind niet gevonden wordt, bijvoorbeeld omdat het kind eerst zonder burgerservicenummer is opgegeven, dan wordt het laatst opgegeven kind zonder burgerservicenummer gecorrigeerd.

Ik heb scenario's toegevoegd die dit tonen voor partner en kind